lib/account/account-manager.js

```diff
@@ -46,24 +104,28 @@
  * @property {string} regId Unhashed registration id sent to client
  */
 
-module.exports = function (accountService, config, hotp, otDate, promise, random) {
+module.exports = function (accountService, base64, config, hotp, otDate, promise, random, secureHash) {
```

Comment:
Caution:  Starting to get long!  Should move things into services or other libraries.